app/user.go

```diff
@@ -999,6 +1009,18 @@ func (a *App) UpdateActive(user *model.User, active bool) (*model.User, *model.A
 		}
 	}
 
+	directChannels, e := a.Srv().Store.Channel().GetDirectChannelsForUser(user.Id)
```

Comment:
s/e/err

Reply:
Didn't get, can you please elaborate?

Reply:
It is a standard code review syntax, which essentially means replace "e" by "err".

Reply:
okay, got it but err is already been initialized & it references *model.AppError

Reply:
Right, in that case, we use `nErr` which stands for normalized error. We have an ongoing campaign which will convert all store level errors to `error`. So all of this will become `err` at a near point in future.